src/classes/GE_GiftEntryController.cls

```diff
@@ -46,16 +46,18 @@ public with sharing class GE_GiftEntryController {
     }
 
     @AuraEnabled
-    public static String makePurchaseCall(String token) {
-        GE_PaymentServices paymentServices = new GE_PaymentServices(token);
-        HttpResponse response = paymentServices.makePurchaseCall();
-
-        // Parse the JSON response...
-        if (response.getStatusCode() != 201) {
-            return JSON.serialize(response.getStatus());
-        } else {
-            return JSON.serialize(response.getBody());
-        }
+    public static String makePurchaseCall(DataImport__c dataImportRecord) {
+        GE_PaymentServices paymentServices = new GE_PaymentServices(dataImportRecord);
+        GE_PaymentServices.ResponseWrapper response = paymentServices.makePurchaseCall();
+
+        return JSON.serialize(response);
+    }
+
+    // TODO: Remove placeholder tokenize call later
+    @AuraEnabled
+    public static String makeTokenizeCall() {
```

Comment:
@dfuller114 - seeing a few of these older methods that shouldn't be in the feature branch anymore.

Reply:
that's likely because of the merge conflicts I'm seeing at the bottom